### PR TITLE
1163 - Start funding not redirected to funding page

### PR DESCRIPTION
**Pull request by hiaux0**
## What was done
- increase timeout
- add `?.` check in `DisposableCollection.ts`

#### After
- should redirect 100% of the time to funding page

### CODE REVIEW

File: src/dealDashboard/dealVotes/dealVotes.ts [hunk 1]
```diff
@@ -101,7 +103,7 @@ export class DealVotes {
     }
 
     if (await this.deal.createSwap()) {
-      await Utils.waitUntilTrue(() => !!this.deal.contractDealId); //have to await this so the contractDealId is populated before redirecting to the funding page
+      await Utils.waitUntilTrue(() => !!this.deal.contractDealId, CREATE_SWAP_TIMEOUT); //have to await this so the contractDealId is populated before redirecting to the funding page
```
Comment (IonelLupu) [inline src/dealDashboard/dealVotes/dealVotes.ts:106]: funded a couple of deals and it seems the error doesn't show up anymore

File: src/services/DisposableCollection.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export class DisposableCollection implements IDisposable {
       this._dispose(disposable);
     } else {
       for (disposable of this.disposables) {
-        disposable.dispose();
+        disposable?.dispose();
```
Comment (IonelLupu) [inline src/services/DisposableCollection.ts:21]: why is this needed?

Comment (hiaux0) [reply src/services/DisposableCollection.ts:21]: `this.disposables` array had `undefined` as values.
I could have investigate why, but this was the easiest fix

**Review by IonelLupu — APPROVED** [review]
